app/components/welcome/welcome.js

```diff
@@ -174,9 +174,9 @@ const dialogs = [
           between.
         </p>
         <p>
-          Hypergraph Vault makes your work available even when your computer is
-          offline. Plus, we're working with libraries on archiving it for future
-          generations! 👵🏾👨🏻👶
+          Hypergraph Vault is our cloud service that makes your work available
```

Comment:
```suggestion
          Hypergraph Vault is our service that makes your work available
```

Cloud has a negative connotation, especially in the P2P space. I'm also not sure this level of technicality is helpful here. Any reasons for specifically mentioning it?

Also, we aren't sure it even is going to be a cloud service, at all. It can run just fine on Chris' laptop, and maybe it will end up in some data centers and never the cloud.

Reply:
I didn't know about the negative connotation. How about "hosting" or "storage" then? I'm looking for simple ways to illustrate the difference between using the vault and not using the vault without literally explaining everything in detail.

For example, during one of last week's tests, it was thought or considered for a while that the vault would also be p2p-based and might require users to offer bandwidth and storage. The word "cloud" came up as something that is nowadays understood as something like "we'll take of your files for you" and implies safety.